assemble/conf/log4j2-service.properties

```diff
@@ -20,7 +20,7 @@
 ## Log4j2 file that configures logging for all Accumulo services
 ## The system properties referenced below are configured by accumulo-env.sh
 
-status = info
+status = warn
```

Comment:
We [deprecated the `status` attribute](https://logging.apache.org/log4j/2.x/manual/configuration.html#configuration-attribute-status) in version `2.24.0`, since this is a per-configuration setting that modifies a resource shared by all the logger context configurations of a system.

We recommend removing the attribute and using the [`log4j2.statusLoggerLevel`](https://logging.apache.org/log4j/2.x/manual/configuration.html#configuration-attribute-status) configuration property.